modules/appeals_api/app/swagger/appeals_api/v2/swagger_sc.json

```diff
@@ -127,6 +127,33 @@
             "schema": {
               "type": "string"
             }
+          },
```

Comment:
Let's not release this as prod documentation just yet. We'll make a separate ticket for putting them up in prod.